mods/pokebattler.php

```diff
@@ -9,6 +9,9 @@
 // Check access.
 bot_access_check($update, 'pokedex');
 
+// Levels available for import at PokeBattler
+$levels = array('6', '5', '3', '1');
```

Comment:
Maybe we should add a POKEBATTLER_RAID_LEVELS or POKEBATTLER_IMPORT_LEVELS or something like that to the default-config and get rid of this here?

Reply:
That would mean removing the whole feature of being able to choose on the fly which ones to import? I'm pretty sure someone would miss it.

Reply:
Let's revisit this one separately so we can all get a working import now. :-)